stock_request/models/stock_request.py

```diff
@@ -340,6 +341,49 @@ def _prepare_procurement_values(self, group_id=False):
     def _skip_procurement(self):
         return self.state != "draft" or self.product_id.type not in ("consu", "product")
 
+    def _prepare_stock_move(self):
+        return {
```

Comment:
This phantom move will be garbage and will corrupt statistics like forecasting, so we should try to get another way. And if not, at least `location_id` and `location_dest_id` should be both `self.location_id.id` in my understanding.

I know that allocations require a move to be linked, but I think we can relax such requirement. The only place where it's used is for checking its state for putting the opened qty.